tutorials/spa-create-excel-automation/spa-create-excel-automation.md

```diff
@@ -0,0 +1,199 @@
+---
+author_name: Ramakrishnan Raghuraman
+author_profile: https://github.com/r3ksk
+title: Build your first automation using Excel SDK of SAP Process Automation
+description: In this tutorial, you will be developing basic automations using Excel SDK, In the end, you will know how to read excel data for a given range using **Excel Cloud Link**, or using traditional approach. In this tutorial, we will show how you can apply filter on the excel data, copy the filtered data onto a new excel sheet and save the file with a new name.
```

Comment:
Use active voice "you will develop"  Capitalize "Excel"
Remove 3rd person plural (we, our, ours)

Reply:
fixed